labs/iot/gps-lab/server/code/application.py

```diff
@@ -0,0 +1,65 @@
+from flask_socketio import SocketIO, emit
```

Comment:
Can we have more comments please to guide new developers

Reply:
> The file `raspberry-pi-pins.jpg` - what is the license for this? I can find it on a medium post without any details on attribution. Can you confirm we are licensed to use this?

I have added a different file from raspberrypi site and it is CC license and I added the source to it as well. Other images are  not from internet.